tt_metal/hw/inc/wormhole/eth_l1_address_map.h

```diff
@@ -23,18 +23,24 @@ struct address_map {
   static constexpr std::int32_t DATA_BUFFER_SIZE_ETH = 4 * 1024;
   static constexpr std::int32_t DATA_BUFFER_SIZE_NOC = 16 * 1024;
   static constexpr std::int32_t DATA_BUFFER_SIZE = 24 * 1024;
+  static constexpr std::int32_t PRINT_BUFFER_SIZE = 224 ;
```

Comment:
This can be taken from `tt-metal/tt_metal/hostdevcommon/common_runtime_address_map.h`